Deployment Workflow - Missing Aspire CLI

```bash
Run aspire deploy \
  aspire deploy \
    --environment Production \
    --clear-cache \
    --include-exception-details
  shell: /usr/bin/bash -e {0}
  env:
    DOTNET_ROOT: /usr/share/dotnet
/home/runner/work/_temp/eb6423b0-0b48-4246-aede-c2ba8033c39d.sh: line 1: aspire: command not found
Error: Process completed with exit code 127.
```

--- a/.github/workflows/cd.yml
+++ b/.github/workflows/cd.yml
@@ -58,6 +58,12 @@ jobs:
       - name: Install Aspire Workload
         run: dotnet workload install aspire
 
+      - name: Install Aspire CLI
+        run: dotnet tool install -g Aspire.Cli
+
+      - name: Validate Aspire CLI Installation
+        run: aspire --version
+
       - name: Azure Login
         uses: azure/login@v2
         with:

